test/dotnet-add-p2p.Tests/GivenDotnetAddP2P.cs

```diff
@@ -13,45 +12,13 @@ namespace Microsoft.DotNet.Cli.Add.P2P.Tests
 {
     public class GivenDotnetAddP2P : TestBase
     {
+        const string TestGroup = "NonRestoredTestProjects";
+        const string ProjectName = "DotnetAddP2PProjects";
```

Comment:
well, this is testing adding references - having them in separate folders is also not good